### PR TITLE
chore: bump lib versions

### PR DESCRIPTION
**Pull request by Betisman**
Bump the versions of both application insight libs this lib uses.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
   },
   "homepage": "https://github.com/guidesmiths/systemic-azure-metrics#readme",
   "dependencies": {
-    "applicationinsights": "^1.4.0",
-    "applicationinsights-native-metrics": "0.0.5",
-    "debug": "^4.1.1"
+    "applicationinsights": "^2.1.3",
```
Comment (kevinccbsg) [inline package.json:17]: Isn't this a breaking change @Betisman ? `1.4.0` -> `2.1.3` 🙄

**Review by kevinccbsg — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (UlisesGascon) [reply package.json:17]: I agree with @kevinccbsg, seems like a breaking change :)

**Comment by UlisesGascon** [comment]
@Betisman @kevinccbsg  do you want me to add the CI for autopublish in NPM in a separate PR?

**Review by kevinccbsg — APPROVED** [review]
After reviewing the breaking change I think it will not affect us. We don't have node < v8 in our projects.
https://github.com/microsoft/ApplicationInsights-node.js/pull/740

**Pull request merged**